docs/the-fleetshard-pattern.md

```diff
@@ -0,0 +1,35 @@
+This document contains useful links on the fleetshard pattern.
+
+# Why the fleetshard model? 
+
+See [ADR-4:Fleet manager - fleet shard local autonomy](https://architecture.appservices.tech/ap/4/)
+
+# I need a fleet manager deployed to get initial feedback but I do not have fleetshard components, what do I do? 
+
+You could still have the first version of the fleet manager that directly pushes (via SyncSet) the CRs to the data plane clusters to deploy your service. For that you could rely on the [create syncset](https://github.com/bf2fc6cc711aee1a0c2a/ffm-fleet-manager-go-template/blob/main/pkg/client/ocm/client.go#L33) and [delete syncset]() methods in the [provisioning reconcilers](../internal/dinosaur/internal/workers/dinosaur_mgrs/provisioning_dinosaurs_mgr.go). The syncset can be updated (when some resources changes) via [update syncset](https://github.com/bf2fc6cc711aee1a0c2a/ffm-fleet-manager-go-template/blob/main/pkg/client/ocm/client.go#L34) or deleted (when a managed instance is to be deprovisioned) via [delete syncset](https://github.com/bf2fc6cc711aee1a0c2a/ffm-fleet-manager-go-template/blob/main/pkg/client/ocm/client.go#L36) methods. From this model you could iterate by building the fleetshard components and integrate them on the next phase. 
```

Comment:
When I was writing this, I was left wondering though if it will be useful for users to have a version of the template that creates a managed instance via syncset. I reckon that this how most teams will likely start before they have the fleetshard components available. 

Thoughts? @davidffrench @miguelsorianod

Reply:
Hmm maybe having the pointers is just enough.

Reply:
I'll create the issue before merging this PR